data/datastore-proto/src/main/proto/com/eblan/launcher/data/datastore/proto/appdrawer/background_color.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
```

Buf (1.72.0)

[error] Files must have a package defined.

(PACKAGE_DEFINED)